ui/src/assets/i18n/de.json

```diff
@@ -105,10 +105,10 @@
     "Loading Data, please wait...": "Lade Daten, bitte warten...",
     "ENV_APP_WARNING": "Bitte erzeuge ein Zertifikat für die Anwendung und die Umgebung unter ",
     "REMAINING_CHARACTERS_TEXT": "{{characters}} verbleibende Zeichen",
-    "Owner of Topics": "Topic-Eigentümer",
-    "User of Topics": "Topic-User",
+    "Owned Topics": "Topic-Eigentümer",
+    "Subscribed Topics": "Topic-Abonnent",
```

Comment:
should be plural right? (=> Abonnenten)

Reply:
Nope, the application is **one** subscriber (Abonnent) of **multiple** topics. So this reads as, "Ist Abonnent von folgenden Topics".